src/rules/color-named/__tests__/index.js

```diff
@@ -286,3 +286,37 @@ testRule(rule, {
     column: 22,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [ "never", { ignore: ["inside-function"] } ],
+
+  accept: [ {
+    code: "a { color: #000; }",
+  }, {
+    code: "a { color: gray(70%); }",
+  }, {
+    code: "a { background-image: url(red); }",
+  }, {
+    code: "a { color: map-get($colour, blue(60%)); }",
+  }, {
+    code: "a { color: map-get($colour, blue); }",
+  } ],
+
+  reject: [ {
+    code: "a { color: map-get($colour, blue), rebeccapurple; }",
```

Comment:
@jeddy3 I get your point. I used this test to make sure that when I make `return false` from `walk` function, the function continues to process the remaining data rather than make full stop. What do you think? Should I remove this test anyway?